apps/consumer-client/src/pages/examples/add-pcd.tsx

```diff
@@ -186,11 +183,12 @@ function AddEthAddrPCDButton(): JSX.Element {
     if (!("ethereum" in window)) {
       alert("Please install MetaMask to use this dApp!");
     } else {
-      const ethereum: ethers.providers.ExternalProvider = window.ethereum;
+      const ethereum: ethers.providers.ExternalProvider =
```

Comment:
We already know that `window.ethereum` exists due to the preceding `if` clause. Is there a reason to think that it could both exist *and* have the value `undefined`?